plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -14,9 +15,9 @@ class Release < ActiveRecord::Base
     validates :status, inclusion: STATUSES
     validate :docker_image_in_registry?, on: :create
 
-    STATUSES.each do |s|
-      define_method("#{s}?") { status == s }
-    end
+    scope :not_dead, -> { where.not(status: :dead) }
+    scope :excluding, ->(ids) { where.not(id: ids) }
+    scope :with_not_dead_release_docs, -> { joins(:release_docs).where.not(Kubernetes::ReleaseDoc.table_name => { status: :dead }) }
```

Comment:
Nop. `dead` represents a old release / release_doc. It's dead, gone, we don't care about it anymore. :)

This is used to reconcile old releases with the Cluster: if there's a new release being deployed, we're fetching and marking as `dead` each previous Release that:
- is not dead
- does not have a pod being shutdown in the cluster